docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,159 @@
+# Percona Monitoring and Management 2.31.0
+
+| **Release date:** | Sep 26, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### Enhaced query building, navigation and Alerting
+
+We have upgraded Grafana to version 9.1 in PMM 2.31.0. With this upgrade, you can get a simplified user interface, enhanced visualizations, and a default unified Alerting experience. For more information, see What’s new in Grafana.
+ 
+The following features have been introduced with PMM 2.31.0:
+ 
+- **Expandable navbar (main menu)**
+ 
+    For an enhanced user experience and better accessibility, the main menu has been expanded with crucial elements such as starred dashboards, system, MongoDB, MySQL, PostgreSQL, ProxySQL, and HAProxy.
+ 
+- **Prometheus Query Builder**
+
+    A new UI query builder has been introduced, by which you can write Prometheus queries effortlessly. Prometheus queries can be complex to grasp, but with this Grafana release in PMM, it becomes easier. 
+
+- **Improvements to dashboard search**
+    
+    The dashboards can now be searched by panel title as well as dashboard title. Also, the overall performance has been enhanced as querying and sorting have become much faster.
+
```

Comment:
can we add 1-line info for : https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v9-0/#command-palette

Reply:
Sure.